.github/workflows/c8run-build.yaml

```diff
@@ -109,6 +148,7 @@ jobs:
     name: C8Run Test Windows
```

Comment:
@jessesimpson36 @hisImminence I wonder why the windows step is not reusing the setup-c8run action, the action actually already has platform specific logic baked in, right?

Reply:
it was either that compiling the camunda core did not work on windows, or one of the actions (setup-build or build-zeebe) was not built for running on windows nodes. I don't recall the exact error.